specs/node_agent_flows_and_namespaces.adoc

```diff
@@ -0,0 +1,151 @@
+// vim: tw=79
+:toc:
```

Comment:
The only point I have at the moment is that it looks very high level to me and with more examples and snippets added from definitions would make it clearer for everybody else.